control/autoware_collision_detector/package.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>autoware_collision_detector</name>
+  <version>0.1.0</version>
+  <description>The collision_detector package</description>
+  <maintainer email="shinnosuke.hirakawa@tier4.jp">Shinnosuke Hirakawa</maintainer>
```

Comment:
Please remove me as a maintainer.

Reply:
Fixed in 80d5e17fd9bd3bf78ee65b990c2a69e5d26a5b9f